consensus/hotstuff/verification/combined_verifier.go

```diff
@@ -122,3 +123,65 @@ func (c *CombinedVerifier) VerifyQC(signers flow.IdentityList, sigData []byte, b
 	}
 	return stakingValid, nil
 }
+
+// VerifyConsensusQC verifies the validity of a combined signature on a quorum certificate.
+func (c *CombinedVerifier) VerifyConsensusQC(signers flow.IdentityList, sigData []byte, block *model.Block) error {
```

Comment:
Who will call this function? Event handler?

Reply:
This function will be removed to replace the existing `VerifyQC` method. I didn't replace now because I don't want to break the existing integration tests. 

Will replace once we swap all the V2 modules and fix the integration tests